src/modules/Ticket/type.ts

```diff
@@ -0,0 +1,23 @@
+export type createTicketDataType = {
```

Comment:
We can follow a consistent way of defining types
So for objects we try to use Interface & the name starts with 'I'
So for this type it would be
```
export interface ICreateTicketPayload {
Styling with Markdown is supported
  title: string;
  description: string;
  category_id: number;
  department_id: number;
  ticket_type: number;
  resolver_id: number;
}
```